cms/envs/common.py

```diff
@@ -357,7 +357,8 @@
     'ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES': True,
     'SHOW_FOOTER_LANGUAGE_SELECTOR': False,
     'ENABLE_ENROLLMENT_RESET': False,
-
+    # Settings for course import olx validation
+    'ENABLE_COURSE_OLX_VALIDATION': True,
```

Comment:
Why this setting is not added in `production.py` as others?

Reply:
Here is the inheritance flow.

`common.py => production.py`

Populating FEATURES for prod is dynamic, and it loads values from the edx-internal.
https://github.com/edx/edx-platform/blob/547642120b6c9e4b92fa77c58fef7756e053800c/cms/envs/production.py#L274-L276